### PR TITLE
Fix Header Search Paths for Pod Header Files

### PR DESCRIPTION
**Pull request by brianhiss**
This addresses the conflicts from Issue #131 .

### CODE REVIEW

**Comment by isnifer** [comment]
Awesome 👍

File: ios/TPSStripe/TPSCardField.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
 //
 
 #import <React/UIView+React.h>
-#import <Stripe/Stripe.h>
+#import "Stripe.h"
```
Comment (dmytrozavgorodniy) [inline ios/TPSStripe/TPSCardField.h]: It will works only with static library, what about framework?

**Comment by dmytrozavgorodniy** [comment]
@brianhiss Hey, check #134, we think we find better solution for this issue. Could you please check this implementation in your app and give results for us.

**Comment by brianhiss** [comment]
@dmytrozavgorodniy Awesome, I'll take a look in the next day and get back to you.

**Comment by isnifer** [comment]
Closed in favor https://github.com/tipsi/tipsi-stripe/pull/134

**Pull request closed without merging**